sound/sequences/00_sound_player.s

```diff
@@ -6986,6 +6986,42 @@ sound_ref .sound_menu_power_meter
 sound_ref .sound_menu_camera_buzz
 sound_ref .sound_menu_camera_turn
 
+// Crash screen error sfx
+
+sound_ref .sound_menu_critical_error
+
+.envelope_critical_error:
+envelope_line 1 32760
+envelope_line 128 0
+envelope_line 500 500
+envelope_goto 4
+
+.layer_critical_error_0:
+layer_note1 3, 0x3f, 115
+layer_end
+
+.layer_critical_error_1:
+layer_note1 15, 0x3f, 115 
+layer_end
+
+.layer_critical_error_2:
+layer_note1 34, 0x3f, 115 
+layer_end
+
+.layer_critical_error_3:
+layer_note1 39, 0x3f, 115 
+layer_end
```

Comment:
Move these below the channel definition, for consistency with the rest of the file. Also given the jumpscare nature of this sound, I'd lower the velocities on these just a bit (like 103 maybe?)

Reply:
It’s not that loud, and is especially less jumpscare than the current sound.